src/components/SpeciesDeliverableTable/index.tsx

```diff
@@ -78,20 +80,20 @@ const SpeciesDeliverableTable = (): JSX.Element => {
               {strings.SPECIES}
             </Typography>
 
-            <Button
-              icon='plus'
-              id='add-species-to-project'
-              label='Add Species to Project'
-              onClick={() => {
-                setOpenedAddSpeciesModal(true);
-              }}
-              priority='secondary'
-              size='medium'
-            />
+            {!isAcceleratorRoute && (
+              <Button
+                icon='plus'
+                id='add-species-to-project'
+                label='Add Species to Project'
+                onClick={() => setOpenedAddSpeciesModal(true)}
+                priority='secondary'
+                size='medium'
+              />
+            )}
```

Comment:
Is it safe to assume all accelerator users have permission to use this feature or are there any permission checks that should be made here?

Reply:
Hmm I'm not sure if all users can add species. We can add condition later if needed